pkg/apis/job.min.io/v1alpha1/register.go

```diff
@@ -0,0 +1,57 @@
+// Copyright (C) 2022, MinIO, Inc.
```

Comment:
```suggestion
// Copyright (C) 2024, MinIO, Inc.
```

Reply:
These need an update but can be done in another PR.

Reply:
sure!